static/compiler.js

```diff
@@ -676,9 +676,9 @@ define(function (require) {
         }
     };
 
-    Compiler.prototype.onCfgViewClosed = function (id, network) {
+    Compiler.prototype.onCfgViewClosed = function (id) {
```

Comment:
Nice spot! I forgot to remove the arg when I move the destroy funcitonality back to the view itself